src/Main/Main.scss

```diff
@@ -0,0 +1,38 @@
+$primary: #17A2B8;
+
+body {
+    min-width: 370px;
+}
+
+.title-box-width {
+    max-width: 100%;
+    width: 650px;
+    height: 200px;
+}
+
+.banner-title {
+    font-size: 50px;
+}
+
+.custom-text-primary {
+    color: $primary;
+}
```

Comment:
1. this class is used for the logo and for the title
2. the only thing in common between those two is the colour
3. you have made a variable for the colour and used it only once
4. it would be cleaner to write the code in two different places - one for the logo and one for the title (in their own respective scss files)
5. currently the code is in Main, but it doesn't belong to Main and instead it belongs to Title and Navbar
6. the variable for the colour should be in [some sort of common place](https://stackoverflow.com/a/65537376)